File: Linguagem C Exercicios/C-Condicional/OPERADORA.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int qntMin = 0;
    double valorTotal = 0;

    printf("Digite a quantidade de minutos: ");
    scanf("%d",&qntMin);

    if (qntMin <= 100){
        valorTotal = 50.00;
    }else{
        valorTotal = ((qntMin - 100) *2.00) + 50.00;
    };

    printf("Valor total a ser pago: R$ %.2lf",valorTotal);

    return 0;
}

File: Linguagem C#/Condicional/Operadora.cs
using System;
using System.Globalization;

namespace Operadora {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;

            int qnt;
            double valorTotal;

            Console.Write("Digite a quantidade de minutos: ");
            qnt = int.Parse(Console.ReadLine());

            if (qnt <= 100) {
                valorTotal = 50.00;

            }else {
                valorTotal = ((qnt - 100) * 2.00) + 50.00;
            };

            Console.WriteLine("Valor a pagar: "+valorTotal.ToString("F2",CI));
        }
    }
}
